genjava/models/jackson.go

```diff
@@ -20,12 +20,29 @@ func NewJacksonGenerator(types *types.Types) *JacksonGenerator {
 	return &JacksonGenerator{types}
 }
 
-func (g *JacksonGenerator) ReadJson(varJson string, typeJava string) string {
-	return fmt.Sprintf(`objectMapper.readValue(%s, new TypeReference<%s>() {})`, varJson, typeJava)
+func (g *JacksonGenerator) Imports(w *sources.Writer) {
```

Comment:
Are these all fully related to Jackson only? It looks like BigDecimal and other java types here are not specific to JSON libs.

Also consider this signature:
`func (g *JacksonGenerator) Imports() []string` which returns array of packages to import.